src/xperm_rules.md

```diff
@@ -74,6 +75,26 @@ Conditional Policy Statements
 | ----------------------- | ----------------------- | ----------------------- |
 | No                      | No                      | No                      |
 
+### Extended Permission Evaluation
+
+Extended permission rules are evaluated as follows:
+
+* If no extended permissions are defined, only the resource-level policy is
+  considered.
+
+* If an extended permission rule is defined, the policy is first evaluated
+  according to the high-level resource policy. For example:
+
+  * If an *allowxperm* rule is defined, extended permissions will only be
```

Comment:
This sort of makes it sound like "allow" is sufficient (and xperm is totally worthless).  Maybe something about "both checks must pass" or similar would be clearer?

Reply:
That's a good point. I've updated this to state that "both the standard AVC checks and the extended permissions must pass". Please let me know if this statement needs any more clarification.